src/components/TransactionSettings/index.tsx

```diff
@@ -142,7 +145,7 @@ export default function TransactionSettings({ placeholderSlippage }: Transaction
     } else {
       try {
         const parsed: number = Math.floor(Number.parseFloat(value) * 60)
-        if (!Number.isInteger(parsed) || parsed < 60 || parsed > 180 * 60) {
+        if (!Number.isInteger(parsed) || parsed < 60 || parsed > THREE_DAYS_IN_SECONDS) {
```

Comment:
should this be dependent on "expert mode"

Reply:
Probably not IMO. The user can already change the deadline outside of expert mode and the default deadline is set to 30 minutes. Pending transactions that are "stale" can also usually be nullified by submitting a new TX with high gas.

Reply:
AFAIK expert mode mostly refers to slippage, not deadline.